README.md

```diff
@@ -4,15 +4,16 @@
 
 Cloud Native Application Bundles (CNAB) are a package format specification that describes a technology for bundling, installing, and managing distributed applications, that are by design, cloud agnostic.
 
-## CNAB Core 1.0 (Final)
+## CNAB Core 1.1.0 (Final)
 
-The CNAB Working Group with the joint approval of the Executive Directors has approved the CNAB Core 1.0 specification for publication. CNAB Core 1.0 is complete.
+The CNAB Working Group with the joint approval of the Executive Directors has approved the CNAB Core 1.1.0 specification for publication. CNAB Core 1.1.0 is complete.
 
-For more information on the approval process, see [the process documentation](901-process.md). Further changes to CNAB Core will be considered for CNAB Core 1.1.
+For more information on the approval process, see [the process documentation](901-process.md). Further changes to CNAB Core will be considered for CNAB Core 1.2.
 
 ### Branch/Tag Structure
 
 - [master](https://github.com/cnabio/cnab-spec) is the current working draft of all specs
+- [cnab-core-1.1.0](https://github.com/cnabio/cnab-spec/tree/cnab-core-1.1.0) is the CNAB Core 1.1.0 final draft
```

Comment:
This tag is yet to be created; the intention is to cut the tag after this PR is approved, merged and included in the forthcoming `cnab-core-1.1.0-ga` branch.  Ref ordering in https://github.com/cnabio/cnab-spec/issues/387